src/main/index.js

```diff
@@ -192,9 +195,7 @@ app.on('ready', () => {
 })
 
 app.on('window-all-closed', () => {
-  if (process.platform !== 'darwin') {
-    app.quit()
-  }
+  app.quit()
```

Comment:
:thinking: might want to discuss this to be sure if this is what we want to have happen on mac

Reply:
Me and Sam discussed it :colbert: it fixes a bug, plugins can't be installed after: closing all windows (not quitting the app) then opening the window again and trying to install a plugin

Reply:
Has anyone looked into that yet? :hmhm: